qa/L0_lifecycle/lifecycle_test.py

```diff
@@ -3566,6 +3566,33 @@ def test_load_new_model_version(self):
         self.assertEqual(server_log.count("[PB model] Loading version 4"), 2)
         self.assertEqual(server_log.count("successfully loaded 'identity_fp32'"), 3)
 
+        # update model config to only load version 4
+        config_path = os.path.join("models", model_name, "config.pbtxt")
+        with open(config_path, mode="r+", encoding="utf-8", errors="strict") as f:
+            config = f.read()
+            config = config.replace(
+                "version_policy: { specific: { versions: [1, 2, 3, 4] } }",
+                "version_policy: { specific: { versions: [4] } }",
```

Comment:
I think this test case is confirming that upon changing version_policy to just version 4 (which is already loaded), no reload is triggered. 

But I think the test case I'm describing is the opposite -- if you change _only_ the version_policy to a version that isn't already loaded - will it load the model, or will the version policy change be ignored when considering reload criteria in core?

Reply:
Does that make sense?

Reply:
I see, a complete load test with only version policy change, added [Expand test to cover load action with only version addition to policy](https://github.com/triton-inference-server/server/pull/7527/commits/3c6980501dd66ee3a575050987e56372d7f5d71a)

Reply:
It is working locally, waiting on CI to rerun the L0_lifecycle with updated test script.

Reply:
Nice, thanks!